pageserver/src/layered_repository.rs

```diff
@@ -2007,6 +2043,7 @@ impl LayeredTimeline {
 
                 if img.len() == page_cache::PAGE_SZ {
                     let cache = page_cache::get();
+                    info!("Put in cache key={:?} lsn={}", key, last_rec_lsn);
```

Comment:
Probably can be `debug!`

Reply:
Sorry, it is just temporary debug message - will remove it.

Reply:
Done